bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncTable.java

```diff
@@ -95,7 +98,8 @@ public BigtableAsyncTable(BigtableAsyncConnection asyncConnection,
       HBaseRequestAdapter hbaseAdapter) {
     this.asyncConnection = asyncConnection;
     BigtableSession session = asyncConnection.getSession();
-    this.client = new BigtableDataClient(session.getDataClient());
+    this.client = new BigtableDataClient(session.getDataClient(), session.getClientWrapper());
```

Comment:
Maybe add a comment that the unwrapped data client is going away, once the wrapper is done

Reply:
We have this pattern all over the place.  I'd rather not comment here only.